gammapy/datasets/map.py

```diff
@@ -573,6 +579,9 @@ def from_geoms(
             "gti", GTI.create([] * u.s, [] * u.s, reference_time=reference_time)
         )
         kwargs["mask_safe"] = Map.from_geom(geom, unit="", dtype=bool)
+        kwargs["livetime_map"] = kwargs["mask_safe"].reduce_over_axes(
```

Comment:
The livetime can be initialized as a non-masked map. Just as for the other qualities the mask will be applied in `.stack()` and `.to_masked()`

Reply:
No, but `Map.from_geom(geom, unit="s")` is simpler I guess..